.github/workflows/cd.yaml

```diff
@@ -25,4 +28,33 @@ jobs:
           npm run lint
       - name: Unit Test
         run: |
-          npm run test
\ No newline at end of file
+          npm run test
+  build-and-push:
+    needs: [test-and-lint]
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: true
+      - name: Build and Publish as 'unstable'
+        if: github.ref == 'refs/heads/master'
+        run: |
+          docker login -u $DOCKER_USERNAME -p $DOCKER_PASSWORD
+          docker build --no-cache -t libero/$REPO_NAME:unstable -t $DOCKER_USERNAME/$REPO_NAME:unstable .
```

Comment:
Don't see any need to build the `libero/$REPO_NAME:unstable` variant, so you could remove this although technically it's a NOOP so not causing any harm.

Reply:
Oddly enough I only added this extra tag in because `.scripts` as complaining it couldn't find a `libero/editor-pdf-service` image to deploy. [I think this requires it to be there](https://github.com/libero/scripts/blob/c450a934ad22270bbd5ac229fe844daceb9351f9/github/retag-and-push.sh#L25). I made a not to investigate why the other repos were working without this.